redaxo/src/addons/structure/lib/api_functions/api_article_copy.php

```diff
@@ -8,17 +8,22 @@ class rex_api_article_copy extends rex_api_function
 {
     public function execute()
     {
+        // The article to copy
         $article_id = rex_request('article_id', 'int');
-        $clang = rex_request('clang', 'int', 1);
+        // The destination category in which the given article will be copied
         $category_copy_id_new = rex_request('category_copy_id_new', 'int');
+
         $user = rex::getUser();
 
         $context = new rex_context([
             'page' => rex_be_controller::getCurrentPage(),
-            'clang' => $clang,
+            'clang' => rex_request('clang', 'int', 1),
```

Comment:
Ich persönlich würde lieber die variable behalten... dann hat man in dieser funktion alle request variablen auf einen blick (das sind ja die parameter nach außen)

Reply:
OK.